### ISSUE TITLE
Unable to login anyhow after update to v9

### ISSUE THREAD

**Issue by ericcotelnu** [owncloud/core#24364]
### Steps to reproduce
1. load welcome page

### Expected behaviour
Being able to login.

### Actual behaviour
Nothing. The JS doesn't get executed at all.

### Server configuration
**Operating system**:
Debian GNU/Linux

**Web server:**
nginx

**Database:**
mariadb/mysql latest stable

**PHP version:**
7.0

**ownCloud version:** (see ownCloud admin page)
  - version: 9.0.1.3

**Updated from an older ownCloud or fresh install:**
update

**Where did you install ownCloud from:**
tarball

**Signing status (ownCloud 9.0 and above):**

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
I *can't* login as anything.

**List of activated apps:**
Enabled:
  - activity: 2.2.1
  - calendar: 1.1
  - comments: 0.2
  - contacts: 1.2.0.0
  - dav: 0.1.6
  - documents: 0.12.0
  - external: 1.2
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_external: 0.5.2
  - files_mv: 0.8.2
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.0
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - mail: 0.4.2
  - notes: 1.1.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - sharelinks: 1.1.0
  - systemtags: 0.2
  - tasks: 0.8.1
  - templateeditor: 0.1
  - updatenotification: 0.1.0
  - updater: 0.6
  - user_external: 0.4

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**
```
<?php
$CONFIG = array (
  'instanceid' => 'STRIPPED',
  'passwordsalt' => 'STRIPPED',
  'trusted_domains' => 
  array (
    0 => 'oc.DOMAIN.org',
    1 => 'DOMAIN.org',
  ),
  'datadirectory' => 'STRIPPED/oc/data',
  'dbtype' => 'mysql',
  'version' => '9.0.1.3',
  'dbname' => 'STRIPPED',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'STRIPPED_',
  'dbuser' => 'STRIPPED',
  'dbpassword' => 'STRIPPED',
  'installed' => true,
  'ldapIgnoreNamingRules' => false,
  'log_rotate_size' => 20971520,
  'forcessl' => true,
  'theme' => '',
  'maintenance' => false,
  'mail_smtpmode' => 'smtp',
  'loglevel' => 0,
  'preview_libreoffice_path' => '/usr/bin/lowriter',
  'secret' => 'STRIPPED',
  'singleuser' => false,
  'forceSSLforSubdomains' => true,
  'mail_from_address' => 'owncloud+noreply',
  'mail_domain' => 'domain.org',
  'mail_smtphost' => 'mail.domain.org',
  'mail_smtpport' => '587',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtpauth' => 1,
  'mail_smtpname' => '',
  'mail_smtppassword' => '',
  'mail_smtpsecure' => 'tls',
  'check_for_working_htaccess' => false,
  'remember_login_cookie_lifetime' => 1296000,
  'session_lifetime' => 86400,
  'session_keepalive' => true,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'memcached_servers' => 
  array (
    0 => 
    array (
      0 => 'localhost',
      1 => 11211,
    ),
  ),
  'asset-pipeline.enabled' => true,
  'activity_expire_days' => 365,
  'appstore.experimental.enabled' => true,
  'trashbin_retention_obligation' => 'auto',
);
```

**Are you using external storage, if yes which one:** 
no

**Are you using encryption:** yes/no
yes

**Are you using an external user-backend, if yes which one:**
no

### Client configuration
**Browser:**
any

**Operating system:**
Linux-based

browser JS console:

SyntaxError: expected expression, got '< 'oc.js:1
ReferenceError: oc_appconfig is not defined

**Comment by Xenopathic** [comment]
Could you try disabling the asset pipeline?

**Comment by Watnuss** [comment]
I can second this issue. Having the same error behaviour including the console error. In have a rather minimal configuration without asset-pipeline setting. So I use the default which I guess is 'false'?

**Comment by RealRancor** [comment]
> PHP version: 7.0

7.0.6? If yes see https://github.com/owncloud/core/pull/24343

**Comment by Watnuss** [comment]
@RealRancor: Thanks for the reference. I indeed use php versaion 7.0.6. I have a look.

**Comment by Watnuss** [comment]
I can confirm that the 3 lines change enables website login and owncloud client functionality again. From my side this issue is resolved. But im not the OP.

**Comment by ericcotelnu** [comment]
OK, I'll try it out. Moment

**Comment by ericcotelnu** [comment]
Canconfirm it works with asset-pipeline disabled. I also included the patch mentioned in #24343

**Comment by DeepDiver1975** [comment]
So we can close this - the fix is in #24343

**Comment by ericcotelnu** [comment]
Thanks
